### ISSUE TITLE
using mayor's campaign websites as partisan ground truth

### ISSUE THREAD

**Issue by markusneumann** [desmarais-lab/govWebsites#4]
I am starting to have more and more doubts about this approach, recently fueled by the fact that I noticed that mayors later re-use their mayoral campaign websites as their campaign websites for running for governor/congress, etc. Basically, there is no guarantee that the current state of their campaign website reflects the way it was during the campaign. At the very least, a lot of mayors often seem to make a 'mission accomplished' post in which they brag about their victory - which obviously wasn't there during the campaign.

Also, the results, while admittedly a lot better when incorporating the mayors of the top 100 cities (see figure 6 (cosine corrplot) and table 13 (bootstrapped lower/upper bounds) in the manuscript), they still don't really prove anything.

**Comment by markusneumann** [comment]
One possible alternative approach might be to compare city websites of partisan mayors to those who are nonpartisan in NY, WA, or OR (since these states have both types).

**Comment by bdesmarais** [comment]
This raises a good point, Markus. We did't have multiple states when we first started thinking about this ground truth test. It would probably serve the end of evaluating ground truth in general to just see if the cosine similarity of dem-dem city websites and rep-rep city websites are significantly higher than that of dem-rep city websites, but only comparing pairs across states to avoid within-state coordination on website developers or any other common exposure effects such as geographic proximity. When you mention nonpartisan mayors, are you saying that some mayors are genuinely nonpartisan?
